lang/src/main/resources/lang/en_GB.lang

```diff
@@ -189,6 +189,8 @@ tag.triggerblock.description=Sets the trigger block/s of the portal. Comma separ
 tag.command.description=Sets a command on the post activation of the portal. Comma separated or multi tag.
 tag.portalEvent.description=Triggers the portal from portal events rather than the player entering the portal. (survival nether portal)
 tag.message.description=Sets the message to be displayed when the portal is activated
+tag.conditions.description=Sets the conditions for the portal to be activated
+tag.conditions.fail= You cannot enter this portal at this time
```

Comment:
"The conditions to activate this portal have not been met" or something not sounding time related :)